plaso/analyzers/hashers/entropy.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""The entropy hasher implementation."""
+
+from __future__ import unicode_literals
+
+import codecs
+import collections
+import math
+
+from plaso.analyzers.hashers import interface
+from plaso.analyzers.hashers import manager
+
+
+class EntropyHasher(interface.BaseHasher):
+  """This class provides entropy hashing functionality."""
+
+  NAME = 'entropy'
+  DESCRIPTION = 'Calculates an entropy digest hash over input data.'
+
+  def __init__(self):
+    """Initializes the entropy hasher."""
+    super(EntropyHasher, self).__init__()
+    self._counter = collections.Counter()
+    self._length = 0
+
+  def GetBinaryDigest(self):
+    """Returns the digest of the hash function as a binary string.
+
+    Returns:
+      bytes: binary string hash digest calculated over the data blocks passed to
```

Comment:
Isn't byte value entropy a real number between 0.0 and 8.0 ?

Reply:
It is, this was part of the interface, and now removed.